src/components/transactions/walletTransactions/walletTransactions.js

```diff
@@ -43,13 +45,17 @@ class WalletTransactions extends React.Component {
       filter,
     });
   }
+  onTransactionRowClick(props) {
+    this.props.history.push(`${routes.wallet.path}?id=${props.value.id}`);
+  }
```

Comment:
I really like how cleanly you can solve this with `onTransactionRowClick` after the split you did in the previous PR.